Django_Blog/blog/views.py

```diff
@@ -1,6 +1,38 @@
 from django.shortcuts import render
-from django.views.generic.base import TemplateView 
+from django.views.generic.base import TemplateView
+from django.views.generic import ListView,DetailView
+from django.views.generic.edit import CreateView
+from .form import CommentForm
+from .models import Author,Blog
 # Create your views here.
 
 class index(TemplateView):
     template_name = "base.html"
+
+class AuthorListView(ListView):
+    model = Author
+    template_name = 'author_list.html'
+
+class AuthorDetailView(DetailView):
+    model = Author
+    template_name = 'author_detail.html'
+    context_object_name = 'author'
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context['blog_list'] = Blog.objects.filter(author=self.object)
```

Comment:
Would you please explain me what this line is doing?

Reply:
Display author which has been published particular blog. only display that author detail

Reply:
I told you to add `related_name` above. Remove this function. Use the `author.blogs.all()` in `author_detail.html` instead of `blog_list`. Here blogs is the `related_name` I told you to add.